styles/index.scss

```diff
@@ -36,3 +36,4 @@ limitations under the License.
 @import "code_prism";
 @import "code_secondary_label";
 @import "table_wrapper";
+@import "vimeo";
```

Comment:
I would move this up in the file, to be with wistia/youtube

Reply:
Done.